scrapy/utils/project.py

```diff
@@ -68,10 +68,7 @@ def get_project_settings():
         settings.setmodule(settings_module_path, priority='project')
 
     # XXX: remove this hack
```

Comment:
I think that `# XXX: remove this hack` should be removed as well.

Reply:
The code actually has 2 hacks,so thought it won't matter. Anyway I am doing what you suggested. Thank you.